espnet2/bin/asr_inference_k2.py

```diff
@@ -519,6 +670,16 @@ def get_parser():
         help="The model path of sentencepiece. "
         "If not given, refers from the training args",
     )
+    group.add_argument("--is_ctc_decoding", type=str2bool, default=True)
```

Comment:
`ctc_decoding` does a bit ambiguous to me.
ctc_viterbi_decoding? ctc_frame_sync_decoding?

Reply:
"ctc_decoding" means the decoding graph only contains "a CTC topology"  and is NOT " TLG" decoding.
Shall I change "--is_ctc_decoding" to "--use_ctc_graph"?

Reply:
I see.
Then, I think ctc decoding is fine. 
Could you add some clarifications in the help?